### DISCUSSION TITLE
[Feature] Memories jump to timeline

### DISCUSSION THREAD

**Discussion by palitu** [immich-app/immich#7329] [Feature Request]
### The feature

_(This is typically from the Web View)_

I use often use the memories most days to reminisce, i really enjoy it.  I have recently added ALL of my old photos to immich, to make it the single repo of everything, which is now 70'000 photos, 1'700 videos for 460Gb (BTW, performance is still great!)

Whilst looking at these memories i see a bunch of rubbish photos and duplicates that need to be cleaned out.  Doing it ad-hoc through memories allows me to do it bit by bit over the next few years, with out it being totally overwhelming.  However, the UI does not give me a way to easily delete a number of photos.

**The new features are/could be:**
- allow standard operations from the memories thumbnails view (as a minimum delete fro my use case)
- Provide a way to jump to the photo in the main timeline view, which will allow me to use standard features.

Thanks!

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by arnolicious** [comment]
I am currently implementing the jump to date functionality in the mobile app, however not on web or the other features :)

https://github.com/immich-app/immich/pull/7323

**Comment by kamzil** [comment]
Related: #5225

**Comment by bo0tzz** [comment]
Duplicate of #5225 / #4115
